tests/test_uility_facade.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
```

Comment:
There is a typo in this file name:
s/uility/utility/

Reply:
fixed